### PR TITLE
Fix apiclient issues

### PR DESCRIPTION
**Pull request by JIeCoRyI3**
Commit: https://github.com/JIeCoRyI3/the-movie-db/commit/4256a3617c11878c4bf0a89d40c6d0b7af7267c8
Fixed apiClient issues. 
Task: https://trello.com/c/GCwjXcwJ/19-fix-api-client-for-work

### CODE REVIEW

File: src/api/apiClient.js [hunk 1]
```diff
@@ -62,15 +62,15 @@ export class ApiClient {
     //  genres
     getAllGenres = async () =>
         await this.postGlobalResource(
-            `/genre/movie/list` + this.createFilterString(),
+            '/genre/movie/list' + this.createFilterString(),
```
Comment (LordGemma) [inline src/api/apiClient.js]: Please use template strings instead of concatenation

File: src/api/apiClient.js [hunk 2]
```diff
@@ -62,15 +62,15 @@ export class ApiClient {
...
             null,
             null,
             'GET',
             true
         );
     filterWithGenres = async (genreObj) =>
         await this.postGlobalResource(
-            `/discover/movie/` + this.createFilterString(genreObj),
+            '/discover/movie/' + this.createFilterString(genreObj),
```
Comment (LordGemma) [inline src/api/apiClient.js]: The same with string concatenation

File: src/api/apiClient.js (hunk 1 above)
Comment (JIeCoRyI3) [reply src/api/apiClient.js]: Fixed

File: src/api/apiClient.js (hunk 2 above)
Comment (JIeCoRyI3) [reply src/api/apiClient.js]: Fixed

File: src/api/apiClient.js (hunk 1 above)
Comment (JIeCoRyI3) [reply src/api/apiClient.js]: Fix commit: https://github.com/JIeCoRyI3/the-movie-db/pull/17/commits/cccd090d6f5479ab7182aaba488c84ea94eab5d2

File: src/api/apiClient.js (hunk 2 above)
Comment (JIeCoRyI3) [reply src/api/apiClient.js]: Fix commit: cccd090

**Pull request merged**